### PR TITLE
ipv6: initialize iid

### PR DESCRIPTION
**Pull request by cgundogan**
This is the suffix version of  `ipv6_addr_init_prefix()` to initialize the iid part only.

### CODE REVIEW

**Comment by cgundogan** [comment]
@authmillenon I added a unit tests for this

File: tests/unittests/tests-ipv6_addr/tests-ipv6_addr.c [hunk 1]
```diff
@@ -574,6 +574,26 @@ static void test_ipv6_addr_init_prefix(void)
...
+
+    const uint8_t iid[] = { 0x13, 0x02, 0x01, 0x00 };
+
+    ipv6_addr_t c =  { {
+            0x00, 0x01, 0x02, 0x03, 0x04, 0x05, 0x06, 0x07,
+            0x08, 0x09, 0x0a, 0x0b, 0x93, 0x02, 0x01, 0x00
+        }
+    };
+
+    ipv6_addr_init_iid(&a, iid, 31);
```
Comment (authmillenon) [inline tests/unittests/tests-ipv6_addr/tests-ipv6_addr.c]: What happens if `a == NULL`? What happens if `iid == NULL`? What happens if `bits > 128`?

Comment (cgundogan) [inline tests/unittests/tests-ipv6_addr/tests-ipv6_addr.c]: it breaks for the NULL cases. bits > 128 will be limited to 128. Using NULL as input parameters makes no sense here. I don't check for this condition within this function and outsource this responsibility to the user of this function. Like it was done with `ipv6_addr_init_prefix()`

Comment (authmillenon) [inline tests/unittests/tests-ipv6_addr/tests-ipv6_addr.c]: Ok. Just wanted to make sure.

**Comment by authmillenon** [comment]
ACK and go.

**Pull request merged**